app/styles/_shame.scss

```diff
@@ -91,3 +101,7 @@ $ember-power-select-focus-outline: solid 2px rgb(16, 108, 200) !default;
     }
   }
 }
+
+#vote-star-filled {
```

Comment:
I would change it to something more specific, like primary-site-icon or something like that

Reply:
Changed to primary-site-indicator